### DISCUSSION TITLE
Welcome to go-scp Discussions!

### DISCUSSION THREAD

**Discussion by bramvdbogaerde** [bramvdbogaerde/go-scp#83] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vitastest007** [comment]
Hi there, Bram. Can't find another way to contact you in private. Is there any opportunity to text you about your repo not public?

**Reply by bramvdbogaerde** [reply]
Sure, please find my e-mail in one of my latest commits.
